src/collections/_documentation/platforms/cocoa/usage.md

```diff
@@ -0,0 +1,51 @@
+---
+title: 'Advanced Usage'
+---
+
+## Capturing uncaught exceptions on macOS
+
+By default macOS application do not crash whenever an uncaught exception occurs. There are some additional steps you have to take to make this work with Sentry. You have to open the applications `Info.plist` file and look for `Principal class`. The content of it which should be `NSApplication` should be replaced with `SentryCrashExceptionApplication`.
```

Comment:
Can we clarify the steps more specifically here? Suggest:

By default, MacOS applications do not crash whenever an uncaught exception occurs. To enable this with Sentry, 
1. Open the application's `Info.plist` file
2. Search for `Principal class` (the entry is expected to be `NSApplication`)
3. Replace `NSApplication` with `SentryCrashExceptionApplication`